changelogs/fragments/598-vxlan_evpn_nvo-fix.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic.vxlans: Fix VXLAN config/facts handling - moving deprecated uri EVPN_NVO to current one VXLAN_EVPN_NVO. (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/598)
```

Please fix the syntax here to avoid Ansible sanity failures and to match the Ansible standard fragment format.


```suggestion
  - sonic_vxlans - Fix VXLAN config and facts handling; moving deprecated uri EVPN_NVO to current one VXLAN_EVPN_NVO (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/598).
```